geth/geth-genesis.json

```diff
@@ -3,42 +3,853 @@
     "chainId": 1,
```

Comment:
After doing a more involved review I realize we are using the `generated-genesis.json` file now. I wonder if we should remove this to prevent confusion? Or we think is still relevant in other use cases?

Reply:
it's not used by lighthouse, but used by other clients - there are parallel work happening in #78.. I think eventually we'll use the same one